go.sum

```diff
@@ -4,8 +4,8 @@ github.com/Microsoft/go-winio v0.6.1 h1:9/kr64B9VUZrLm5YYwbGtUJnMgqWVOdUAXu6Migc
 github.com/Microsoft/go-winio v0.6.1/go.mod h1:LRdKpFKfdobln8UmuiYcKPot9D2v6svN5+sAH+4kjUM=
 github.com/ProtonMail/go-crypto v0.0.0-20230717121422-5aa5874ade95 h1:KLq8BE0KwCL+mmXnjLWEAOYO+2l2AE4YMmqG1ZpZHBs=
 github.com/ProtonMail/go-crypto v0.0.0-20230717121422-5aa5874ade95/go.mod h1:EjAoLdwvbIOoOQr3ihjnSoLZRtE8azugULFRteWMNc0=
-github.com/WavefrontHQ/go-wavefront-management-api/v2 v2.1.1 h1:Y2XZN5MoPOKIX7YVfAVFOnWYDkAQAh/65Y97Tt4Ez8g=
-github.com/WavefrontHQ/go-wavefront-management-api/v2 v2.1.1/go.mod h1:yOfesauybnZn9yYaovaLRe3RxlsLSRIswmHayvJO9O8=
+github.com/WavefrontHQ/go-wavefront-management-api/v2 v2.2.1 h1:9RWLxiaJ2NsKOGzLBEEDYBEQfqdAcELcIfiz2ETchp4=
```

Comment:
Relatively minor comment but when bumping the minor I would reset the patch to 0. e.g. v2.2.0 

Also possible the specific change is also more of a patch? Hard to say, this repo is leveling up fast because so much is broken / different from the server. :/)

Reply:
(As an aside, seems weird that I can't strike through text in my messages. :( )

Reply:
Yeah, I left out the update to the version file in the v2.2.0 release 😅 So I created an additional patch release to include them